enterprise/frontend/src/metabase-enterprise/whitelabel/lib/whitelabel.js

```diff
@@ -3,6 +3,12 @@ import MetabaseSettings from "metabase/lib/settings";
 
 export function updateColors() {
   const scheme = MetabaseSettings.get("application-colors");
+
+  // MetabaseSettings is not populated in the embedding sdk.
+  if (!scheme) {
+    return;
+  }
+
```

Comment:
For context, settings are populated into this `MetabaseSettings` object from server on page load, but this is not the case for SDKs: https://github.com/metabase/metabase/blob/master/resources/frontend_client/index_template.html#L27